library/src/main/java/com/owncloud/android/lib/common/Quota.kt

```diff
@@ -45,4 +43,8 @@ data class Quota(
     val quota: Long = 0,
 ) : Parcelable {
     constructor(quota: Long) : this(0, quota = quota)
+
+    override fun toString(): String {
+        return "Quota: $quota, free: $free, used: $used, total: $total, relative: $relative"
+    }
```

Comment:
Unneeded unless you want that specific format. `data class`already autogenerates `toString()`